docs/community/SIG-Governance.md

```diff
@@ -0,0 +1,45 @@
+# OCM SIG Governance Model
+
+This document defines the governance framework for Special Interest Groups (SIGs) in the Open Component Model (OCM) project. It is designed to be lightweight and scalable for a small open source project, while ensuring transparency, consistency, and community alignment.
+
+## 1. Purpose
+
+SIGs are groups of contributors focused on specific technical or community topics within OCM. The governance model ensures clear roles, decision-making, and lifecycle management for all SIGs.
+
+## 2. Roles & Responsibilities
+
+- **Chair**: Facilitates meetings, represents the SIG, ensures process adherence. (At least one Chair is required.)
+- **Tech Lead**: Guides technical direction, reviews contributions, supports the Chair. (At least one Tech Lead is required.)
```

Comment:
I think for a small project we dont need differing people for Chair / Tech Lead so im thinking of merging or allowing a seating of the same person on both roles. WDYT?